app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -24,3 +24,34 @@ export const NOTES_TABLE_COLUMN_DATA = [
     width: "70%",
   },
 ];
+
+export const NOTES = [
+  {
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    execution_status: "Getting Started",
+    status: "Created",
+    created_at: "2 hours ago",
+  },
+  {
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    execution_status: "Getting Started",
+    status: "Drafted",
+    created_at: "2 hours ago",
+  },
+  {
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
```

Comment:
Why is this a template string when you aren't performing any interpolation?

Reply:
Both ' and " were used in the sentence that's why I used it.

Reply:
@nancy-binani Can you go through the text and see if there's an actual need for the double quotes(`"`)? Does it add any value to that sentence? Lemme know your thoughts.

Reply:
Fixed the code.

Reply:
Once all changes are made and once you've verified make sure to add a new comment asking for my review. Thanks.